x/mm2/types/msg.go

```diff
@@ -54,3 +57,31 @@ func (m *MsgParams) ValidateBasic() error {
 
 	return m.Params.ValidateBasic()
 }
+
+// ValidateBasic determines whether the information in the message is formatted correctly, specifically
+// whether the signer is a valid acc-address.
+func (m *MsgRemoveMarketAuthorities) ValidateBasic() error {
+	// validate signer address
+	if _, err := sdk.AccAddressFromBech32(m.Admin); err != nil {
+		return err
+	}
+
+	if len(m.RemoveAddresses) == 0 {
+		return fmt.Errorf("addresses to remove cannot be nil")
+	}
+
+	seenAuthorities := make(map[string]struct{}, len(m.RemoveAddresses))
+	for _, authority := range m.RemoveAddresses {
+		if _, seen := seenAuthorities[authority]; seen {
+			return fmt.Errorf("duplicate address %s found", authority)
```

Comment:
Not covered by tests.

Reply:
https://github.com/skip-mev/slinky/pull/293/commits/3754c0555ca61a38ac96d44d74bc48c5a37837d2